### ISSUE TITLE
Proper Documentation

### ISSUE THREAD

**Issue by douira** [tmshl/raspWatch#12]
Eventually, a proper documentation should be generated from the code comments for all parts of the project's code.

**Comment by tmshl** [comment]
I propose to use [phpDocumentor](https://www.phpdoc.org/). Please use the tags described [here](https://www.phpdoc.org/docs/latest/index.html) in the code. Unfortunately, I am not sure if this already supports php7. Maybe we should just wait for the next version - the documentation does not seem to be  high priority right now.

**Comment by douira** [comment]
Yes, good idea. Docs are useful but not essentially necessary right now. Added accompanying tag.
